src/frontends/tensorflow_common/include/helper_ops/internal_operation.hpp

```diff
@@ -35,6 +35,14 @@ class DecoderFake : public ov::frontend::DecoderBase {
                                       "Internal error: the get_input_node method of the fake node decoder is invoked.");
     }
 
+    void get_input_node(size_t input_port_idx,
+                        std::string& producer_name,
+                        size_t& producer_output_port_index,
+                        const OpTypeByName& op_type_by_name) const override {
+        FRONT_END_OP_CONVERSION_CHECK(false,
```

Comment:
Not sure I understand why you use macros FRONT_END_OP_CONVERSION_CHECK with overall meaning "not implemented" in this place, but FRONT_END_NOT_IMPLEMENTED later.